crypto/sha/asm/sha1-x86_64.pl

```diff
@@ -507,12 +507,12 @@ ()
 sha1_block_data_order_ssse3:
 _ssse3_shortcut:
 .cfi_startproc
+	push	%rbp
+.cfi_adjust_cfa_offset -8
```

Comment:
But also without win64 when I step thru this function with gdb
on linux the frame unwinding does not longer work, with this cfi statement.